src/stk500v2.c

```diff
@@ -2549,6 +2553,7 @@ static int stk500v2_paged_load(PROGRAMMER * pgm, AVRPART * p, AVRMEM * m,
             progname, p->desc);
     return -1;
   }
+  memset(cmds, 0, sizeof cmds);
```

Comment:
Perhaps use `sizeof(cmd)` instead of `sizeof cmd` to match the style in the rest of the file?